packages/blocks/src/store/selectors.js

```diff
@@ -296,6 +641,28 @@ export function hasBlockSupport( state, nameOrType, feature, defaultSupports ) {
  * @param {(string|Object)} nameOrType Block name or type object.
  * @param {string}          searchTerm Search term by which to filter.
  *
+ * @example
+ * ```js
+ * import { __, sprintf } from '@wordpress/i18n';
+ * import { store as blocksStore } from '@wordpress/blocks';
+ * import { useSelect } from '@wordpress/data';
+ *
+ * const ExampleComponent = () => {
+ *     const blockNameIsFound = useSelect( ( select ) =>
+ *         select( blocksStore ).isMatchingSearchTerm( 'core/navigation' )
```

Comment:
I think that the second param is missing here. You have to pass the search term to validate it against a given block type.

Reply:
Updated to provide a much clearer example and explain a little better where the search is happening.